app/services/LocationService.js

```diff
@@ -224,7 +220,7 @@ export default class LocationServices {
               return data;
             })
             .catch(error => {
-              console.error('[ERROR] ' + error);
+              console.error('[ERRORrrr] ' + error);
```

Comment:
why did you put "rrr" ??
```suggestion
              console.error('[ERROR] ' + error);
```

Reply:
It was just a test thing, forgot to delete it